src/Arcus.Testing.Security.Providers.InMemory/Arcus.Testing.Security.Providers.InMemory.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFrameworks>net6.0;netcoreapp3.1;netstandard2.1</TargetFrameworks>
```

Comment:
Is it correct to still have .NET Core here ?

Reply:
Maybe we should just consider supporting .NET 6 then ?

Reply:
Ah, that's a good one. Yeah, maybe we shouldn't even concern ourselves with backwards compatibility here. Good one. 👍

Reply:
I'm wondering, though, we mention this in-memory secret provider as a way of easily testing your secret store setup in the feature docs of the security repo... maybe it's not a bad idea to have it still support .NET Core, for a while.

Reply:
Ok for me.